test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/RazorPageExecutionInstrumentationWebSite.Home.ViewWithPartial.html

```diff
@@ -5,23 +5,23 @@
 <p class="class">partial-content</p>
 <p class="class">partial-content</p>
 </div>
-        <p>/Views/_ViewStart.cshtml: Non-literal at 96 contains 16 characters.</p>
-        <p>/Views/_ViewStart.cshtml: Literal at 112 contains 2 characters.</p>
-        <p>/Views/Home/ViewWithPartial.cshtml: Literal at 0 contains 27 characters.</p>
-        <p>/Views/Home/ViewWithPartial.cshtml: Non-literal at 28 contains 39 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 31 contains 2 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 33 contains 8 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Non-literal at 41 contains 4 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 45 contains 1 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 46 contains 20 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 67 contains 2 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 31 contains 2 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 33 contains 8 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Non-literal at 41 contains 4 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 45 contains 1 characters.</p>
-        <p>/Views/Home/_PartialView.cshtml: Literal at 46 contains 20 characters.</p>
-        <p>/Views/_Layout.cshtml: Literal at 82 contains 7 characters.</p>
-        <p>/Views/_Layout.cshtml: Non-literal at 90 contains 12 characters.</p>
-        <p>/Views/_Layout.cshtml: Literal at 102 contains 2 characters.</p>
-        <p>/Views/_Layout.cshtml: Non-literal at 105 contains 12 characters.</p>
-        <p>/Views/_Layout.cshtml: Literal at 117 contains 10 characters.</p>
+/Views/_ViewStart.cshtml: Non-literal at 96 contains 16 characters.
+/Views/_ViewStart.cshtml: Literal at 112 contains 2 characters.
+/Views/Home/ViewWithPartial.cshtml: Literal at 0 contains 27 characters.
+/Views/Home/ViewWithPartial.cshtml: Non-literal at 28 contains 39 characters.
+/Views/Home/_PartialView.cshtml: Literal at 31 contains 2 characters.
+/Views/Home/_PartialView.cshtml: Literal at 33 contains 8 characters.
+/Views/Home/_PartialView.cshtml: Non-literal at 41 contains 4 characters.
+/Views/Home/_PartialView.cshtml: Literal at 45 contains 1 characters.
+/Views/Home/_PartialView.cshtml: Literal at 46 contains 20 characters.
+/Views/Home/ViewWithPartial.cshtml: Literal at 67 contains 2 characters.
```

Comment:
The existing code was incorrect - we don't have 67 characters in `_PartialView`. Additionally the line mappings for `_Layout` were also incorrect. I didn't spend time investigating why it was incorrect.